config.go

```diff
@@ -139,6 +158,17 @@ func ParseConfig(path string) (*Config, error) {
 	// Store a reference to the path where this config was read from
 	config.Path = path
 
+	// Parse the MaxStale component
+	if raw := config.MaxStaleRaw; raw != "" {
+		stale, err := time.ParseDuration(raw)
+
+		if err == nil {
+			config.MaxStale = stale
+		} else {
+			errs = multierror.Append(errs, fmt.Errorf("max_stale invalid: %v", err))
```

Comment:
@armon this is parsed as a time.Duration, so I think the user needs to specify units, don't they? Like `-max-stale 10s` or `max_stale = "10s"`. I don't think Go will let you put a unit-less value here.